includes/class-evf-emails.php

```diff
@@ -122,6 +129,16 @@ public function __construct() {
 	 * @param mixed  $value
 	 */
 	public function __set( $key, $value ) {
+
+		if ( 'form_data' === $key ) {
```

Comment:
I think it's not the right way to add custom code on a Setters function. Please rethink on this and find an alternative way.

Reply:
Fixed 👍